python/test/test_cpp_algos.py

```diff
@@ -355,7 +355,15 @@ def test_louvain_clustering():
 def test_leiden_clustering():
     graph = Graph(get_rdg_dataset("rmat10_symmetric"))
 
-    leiden_clustering(graph, "value", "output")
+    leiden_clustering(graph, "value", "output", True)
+
+    leiden_clustering_assert_valid(graph, "value", "output")
+
+    LeidenClusteringStatistics(graph, "value", "output")
+
+    graph = Graph(get_rdg_dataset("rmat10"))
+
```

Comment:
Can you please add assertion which compares the output of both runs similar to what I added in k-core?